app/forms/citizens/other_assets_form.rb

```diff
@@ -0,0 +1,108 @@
+module Citizens
+  class OtherAssetsForm
+    include BaseForm
+
+    form_for OtherAssetsDeclaration
+
+    attr_accessor :second_home_value,
+                  :second_home_mortgage,
+                  :second_home_percentage,
+                  :second_home_checkbox,
+                  :timeshare_value,
+                  :land_value,
+                  :jewellery_value,
+                  :vehicle_value,
+                  :classic_car_value,
+                  :money_assets_value,
+                  :money_owed_value,
+                  :trust_value
+
+    before_validation :normalize_currency_values, :normalize_percentage_values
+
+    validates :second_home_value, numericality: { greater_than_or_equal_to: 0, if: :any_second_home_value_present? }, allow_blank: true
+    validates :second_home_mortgage, numericality: { if: :any_second_home_value_present? }, allow_blank: true
+    validates :second_home_percentage, numericality: { greater_than_or_equal_to: 0, if: :any_second_home_value_present? }, allow_blank: true
+    validate :all_second_home_values_present_if_any_are_present
+
+    validates :timeshare_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :land_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :jewellery_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :vehicle_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :classic_car_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :money_assets_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :money_owed_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+    validates :trust_value, numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
+
+    CHECKBOX_STATUS_REGEX = Regexp.new(/_checkbox_status\z/)
+
+    def respond_to_missing?(meth, *args)
+      if CHECKBOX_STATUS_REGEX.match?(meth.to_s)
+        true
+      else
+        super
+      end
+    end
```

Comment:
A guard clause would be good here:
```ruby
def respond_to_missing?(meth, *args)
  return true if CHECKBOX_STATUS_REGEX.match?(meth.to_s)
  super
end
```

Reply:
yep.
Out of interest, how do we feel about `CHECKBOX_STATUS_REGEX.match?(meth.to_s) ? true : super`

Reply:
That's good too